src/or/entrynodes.c

```diff
@@ -407,6 +404,13 @@ get_remove_unlisted_guards_after_days(void)
                                  DFLT_REMOVE_UNLISTED_GUARDS_AFTER_DAYS,
                                  1, 365*10);
 }
+
+static time_t
```

Comment:
Please add the trivial func doc here.

Reply:
Fixed - 9a3d655a2a1a8068b25b35494f00ceb805977b6d.